src/Lykke.Service.Nem.SignService/Services/NemSignService.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using io.nem1.sdk.Model.Accounts;
+using io.nem1.sdk.Model.Transactions;
+using Lykke.Service.BlockchainApi.Contract.Wallets;
+using Lykke.Service.BlockchainApi.Sdk;
+
+namespace Lykke.Service.Nem.SignService.Services
+{
+    public class NemSignService : IBlockchainSignService
+    {
+        readonly Address _hotWalletAddress;
+
+        public NemSignService(string hotWalletAddress)
+        {
+            _hotWalletAddress = Address.CreateFromEncoded(hotWalletAddress);
+        }
+
+        public Task<WalletResponse> CreateWalletAsync()
+        {
+            return Task.FromResult(new WalletResponse
+            {
+                PublicAddress = $"{_hotWalletAddress.Plain}${Guid.NewGuid()}",
```

Comment:
Format guid with "N", to exclude "-" characters from the string. This will make copying of the extension more comfortable

Reply:
Makes sense. Changed extension format